pkg/liqoctl/status/info.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019-2022 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package status
+
+import (
+	"fmt"
+	"strings"
+)
+
+// InfoData implements the InfoInterface interface.
+// holds information about a field using a key/value pair.
+type InfoData struct {
+	key   string
+	value []string
+}
+
+// StringIndented return the  String() output but with indentation.
+func (id *InfoData) StringIndented(indent int) string {
+	indentation := strings.Repeat("\t", indent+1)
+	if len(id.value) == 1 {
+		return fmt.Sprintf("%s: %s%s%s", id.key, byellow, id.value[0], reset)
+	}
+	msg := fmt.Sprintf("%s:", id.key)
+	for _, v := range id.value {
+		msg += fmt.Sprintf("\n%s- %s%s%s", indentation, byellow, v, reset)
+	}
+	return msg
+}
+
+// StringNoColor return the  String() output but without colors.
```

Comment:
```suggestion
// StringNoColor returns the String() output but without colors.
```

Same for all occurrences.